scalableminds/webknossos
# When reloading layer, also close its meshfiles

(Analogous to the agglomerate files)

### URL of deployed dev instance (used for testing):
- https://reloadclosemeshfiles.webknossos.xyz

### Steps to test:
- Open a dataset with meshfile, load precomputed mesh, hit reload layer button
- backend logging should show that a meshfile was closed.

### Issues:
- fixes #8074

------
